index.js

```diff
@@ -413,6 +444,34 @@ class MillerColumnsElement extends HTMLElement {
             this.taxonomy.topicClicked(topic)
             topic.checkbox.dispatchEvent(new Event('click'))
             break
+          case 'ArrowUp':
+            event.preventDefault()
+            if (topic.previous) {
+              this.showCurrentColumns(topic.previous)
+              this.focusTopic(topic.previous)
+            }
+            break
+          case 'ArrowDown':
+            event.preventDefault()
+            if (topic.next) {
+              this.showCurrentColumns(topic.next)
+              this.focusTopic(topic.next)
+            }
+            break
+          case 'ArrowLeft':
+            event.preventDefault()
+            if (topic.parent) {
+              this.showCurrentColumns(topic.parent)
+              this.focusTopic(topic.parent)
+            }
+            break
+          case 'ArrowRight':
+            event.preventDefault()
+            if (topic.children && topic.children.length) {
```

Comment:
Are we ever expecting `children` to be not-null and not an array? I think we could simplify to:

```js
if (topic.children) {
```

Reply:
Indeed it seems unnecessary.